04.ls/wc.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, paths = receive_command_argument
+  paths.empty? ? output(count_stdin, options) : output(calculate_file_text(paths), options)

```

Comment:
```suggestion
  file_properties = paths.empty? ? count_stdin : calculate_file_text(paths)
  output(file_properties, options)
```

とした方がよりDRYになります。
あと、同じようなデータが返ってくるはずなのに、countとcalculateで動詞が異なるのは少し気持ち悪いですね。
動詞を揃えた方がいいかも、と思いました

Reply:
DRYにし、動詞を揃えました。